spec/tail_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'filewatch/tail'
 require 'stud/temporary'
+Thread.abort_on_exception = true
```

Comment:
if this test is run with other it will set `abort_on_exception` to `true` forever.

what do you think of 

```ruby
before :all 
  # save the current value
  # set to true
end

after :all
  # restore old value
end
```